### ISSUE TITLE
Bug: GUI - Issue when scrolling the users' table

### ISSUE THREAD

**Issue by yanivbh1** [memphisdev/memphis#943]
### Describe the bug

Hey,
When trying to scroll the users' table within the "Users" page all the way down there is a "glitch."
Record attached.

https://github.com/memphisdev/memphis/assets/70286779/8b58d361-71d7-4099-a656-a14b6c970c8b

### Steps to reproduce

1. Within the GUI, Go to "Users"
2. Fill in at least a "screen height" amount of users
3. Try to scroll the table all the way down

### Affected services

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by yanivbh1** [comment]
@Avitaltrifsik good first issue as well. Perhaps we have a contributor who would like to help here

**Comment by akarsh-jain-790** [comment]
@yanivbh1 I would like to work on this.

**Comment by yanivbh1** [comment]
Thank you, @akarsh-jain-790 .
@Avitaltrifsik is this one available?

**Comment by Avitaltrifsik** [comment]
@yanivbh1 Yes! @akarsh-jain-790 Would you like me to assign you to this one?

**Comment by avrhamNeeman** [comment]
We made some improvements to the user management, and it fixed the issue.
Thank you @akarsh-jain-790!
